src/conf/core-site.xml

```diff
@@ -55,4 +55,10 @@ deployments this is useful as you can use relative paths eg ./bin/dl -ls / -->
     <name>fs.dl.impl.transportScheme</name>
     <value>https</value>
   </property>
+
+  <!-- The location of the AES key. The file should be exactly 16 bytes long. -->
```

Comment:
Also, I think having an extra option to enable/disable encryption would be a bit of an overkill, as this can easily be controlled by commenting/uncommenting the fs.dl.impl.encryptionKeyPath property.